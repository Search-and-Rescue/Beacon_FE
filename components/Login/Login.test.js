describe('Home', () => {
  it.skip('should', () => {

  })
})