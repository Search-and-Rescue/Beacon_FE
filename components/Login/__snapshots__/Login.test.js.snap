// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should match the snapshot with all of the data passed through 1`] = `
<Component
  hardwareAccelerated={false}
  visible={true}
>
  <View
    style={Object {}}
  >
    <TextInput
      allowFontScaling={true}
      placeholder="Email"
      rejectResponderTermination={true}
      style={Object {}}
      underlineColorAndroid="transparent"
    />
    <TextInput
      allowFontScaling={true}
      placeholder="Password"
      rejectResponderTermination={true}
      style={Object {}}
      underlineColorAndroid="transparent"
    />
    <View
      style={Object {}}
    >
      <Button
        title="login"
      />
    </View>
  </View>
</Component>
`;
